### ISSUE TITLE
Dark theme is not applied around `input`/`textArea`

### ISSUE THREAD

**Issue by kittaakos** [poychang/github-dark-theme#278]
See the attached images. Note, the markdown preview is also broken :( 
The GH guys must have changed something:

![Screen Shot 2020-10-26 at 09 04 25](https://user-images.githubusercontent.com/1405703/97147863-7f05f080-176a-11eb-9ebb-19874d01e62c.jpg)
![Screen Shot 2020-10-26 at 09 03 54](https://user-images.githubusercontent.com/1405703/97147870-81684a80-176a-11eb-90c3-0ccb98ef592f.jpg)
![Screen Shot 2020-10-26 at 09 07 02](https://user-images.githubusercontent.com/1405703/97147983-a9f04480-176a-11eb-9ecd-b4968bdce233.jpg)

Thank you for the great lib ❤️

**Comment by RedSparr0w** [comment]
Are you running `v1.1.16`?
As I believe this was fixed in #275

**Comment by tariq86** [comment]
It's fixed in the latest release, but it looks like the latest version available for Firefox is 1.1.13:

![no-updates-found](https://user-images.githubusercontent.com/646561/97368137-4d7b4b00-1878-11eb-8fa9-528884e06e9c.png)

**Comment by kittaakos** [comment]
> Are you running `v1.1.16`?

Yes. I was running with `1.1.16`. It works with `1.1.17` in Brave. 🎉 

I'd propose closing this issue and track the FF-specific version thing separately but it's the maintainer's call 😊

**Comment by poychang** [comment]
The Firefox extension stopped at version 1.1.13 because the package platform blocked my release.  They require uploading the source code to release the package.  Even this is open source project. Currently I can only upload manually, if anyone knows how to upload the source code automatically, please let me know, thank you very much.

**Comment by RedSparr0w** [comment]
I thought you only need to upload the source code if the code is minified?
But it looks like the FireFox version already has minification disabled..

**Comment by poychang** [comment]
@RedSparr0w Yes, it's already disable minification for firxfox. But according [this](https://extensionworkshop.com/documentation/publish/source-code-submission/), if use build tools like browserify or webpack, must upload source code. That's the issue.

**Comment by foss-** [comment]
Can you do a manual upload once so that a recent version is available on Firefox Addons and then tackle automatic updates in a separate issue?

**Comment by poychang** [comment]
@foss- I did that. Wait for approval from them.
